docs/configuration/identity-providers/oidc.md

```diff
@@ -206,23 +212,25 @@ required: no
 The maximum lifetime of an ID token. For more information read these docs about [token lifespan].
 
 ### refresh_token_lifespan
+
 <div markdown="1">
 type: string
-{: .label .label-config .label-purple } 
+{: .label .label-config .label-purple }
 default: 30d
 {: .label .label-config .label-blue }
 required: no
 {: .label .label-config .label-green }
 </div>
 
-The maximum lifetime of a refresh token. This should typically be slightly more the other token lifespans. This is 
```

Comment:
Not sure why the comment about how long the token should be was removed. However I can just add it back later in another paragraph. Other than that LGTM!